pymc/distributions/transforms.py

```diff
@@ -266,6 +271,66 @@ def bounds_fn(*rv_inputs):
         super().__init__(args_fn=bounds_fn)
 
 
+class ZeroSumTransform(RVTransform):
+    """
+    Constrains the samples of a Normal distribution to sum to zero
```

Comment:
The transform isn’t limited to normal distributions. I think that it’s best to take that out of the docstring and just say that it constrain the samples to sum to zero

Reply:
Done